ballerina/tests/mock_service.bal

```diff
@@ -0,0 +1,224 @@
+// Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+
+import ballerina/http;
+import ballerina/log;
+
+listener http:Listener httpListener = new (9090);
+
+http:Service mockService = service object {
+    
+
+    # Remove a bookmarked Post
+    #
+    # + id - The ID of the authenticated source User whose bookmark is to be removed.
+    # + tweet_id - The ID of the Post that the source User is removing from bookmarks.
+    # + return - returns can be any of following types 
+    # http:Ok (The request has succeeded.)
+    # http:Response (The request has failed.)
+    resource function delete users/[UserIdMatchesAuthenticatedUser id]/bookmarks/[TweetId tweet_id]() returns BookmarkMutationResponse|http:Response {
+        BookmarkMutationResponse response = {
+            "data":{"bookmarked":false}
+        };
+        return response;
```

Comment:
We can merge these 2 statements as follows. Let's apply the same to all the mock resource functions
```suggestion
        return {
            "data":{"bookmarked":false}
        };
```